guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -5,8 +5,9 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 
 .Prerequisites
 ifdef::satellite[]
-* Review Known Issues before you begin an upgrade.
-For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Review known issues before you begin an upgrade.
+** {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
```

Comment:
It looks like this is the link that will take users directly to the Known issues chapter. (Also, it really is named `Known issues`, lowercase.)

```suggestion
** {ReleaseNotesDocURL}known-issues[Known issues in {ProjectName} {ProjectVersion}].
```

Reply:
@AkshayGadhaveRH This should be incorporated.

Reply:
Sorry, missed it! Thanks @Lennonka